benchmark/mmitt.py

```diff
@@ -0,0 +1,61 @@
+# Load CSV (using python)
+import csv,math,copy
+import numpy as np
+import mmit
+from mmit import MaxMarginIntervalTree
+from mmit.pruning import min_cost_complexity_pruning
+from mmit.model_selection import GridSearchCV
+from mmit.metrics import mean_squared_error
+
+o = []
+filename = ['/home/parismita/mmit_data/auto93/features.csv', '/home/parismita/mmit_data/auto93/targets.csv', 
+			'/home/parismita/mmit_data/autohorse/features.csv', '/home/parismita/mmit_data/autohorse/targets.csv',
+			'/home/parismita/mmit_data/autompg/features.csv', '/home/parismita/mmit_data/autompg/targets.csv',
+			'/home/parismita/mmit_data/autoprice/features.csv', '/home/parismita/mmit_data/autoprice/targets.csv',
+			'/home/parismita/mmit_data/baskball/features.csv', '/home/parismita/mmit_data/baskball/targets.csv',
+			'/home/parismita/mmit_data/bodyfat/features.csv', '/home/parismita/mmit_data/bodyfat/targets.csv',
+			'/home/parismita/mmit_data/cloud/features.csv', '/home/parismita/mmit_data/cloud/targets.csv',
+			'/home/parismita/mmit_data/cpu/features.csv', '/home/parismita/mmit_data/cpu/targets.csv',
+			'/home/parismita/mmit_data/meta/features.csv', '/home/parismita/mmit_data/meta/targets.csv' ,
+			'/home/parismita/mmit_data/sleep/features.csv', '/home/parismita/mmit_data/sleep/targets.csv']
+
+
+for i in xrange(20):
```

Comment:
You could have simply used this class to load the data: https://github.com/tdhock/mmit-paper/blob/master/mmit.predictions.py#L23